pkgs/by-name/gi/gitbutler/package.nix

```diff
@@ -0,0 +1,88 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, makeDesktopItem
+, gitbutler-ui
+, copyDesktopItems
+, wrapGAppsHook
+, pkg-config
+, perl
+, jq
+, moreutils
+, libsoup
+, webkitgtk
+, glib-networking
+, libayatana-appindicator
+, git
+}:
+
+let
+  version = "0.10.11";
+in
+assert lib.assertMsg (version == gitbutler-ui.version) "The GitButler version does not match the GitButler UI version!";
```

Comment:
```suggestion
```
if we inherit the version anyway, then there is not really a point doing this

Reply:
I think it might be useful to prevent improper overrides. But the current version will not do that, as you point out.